### PR TITLE
River login page

### PR DESCRIPTION
**Pull request by esump**
-Login page modeled after other settings pages
-Has sign in/sign out buttons and shows profile picture
-Sign out is not really functional

### CODE REVIEW

File: src/pages/settings/login.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            </table>
+          </div>
+        </div>
+        <Footer currentPage="settings"/>
+      </div>
+    </div>
+
+  )
+
+}
```
Comment (michael-hewitt) [inline src/pages/settings/login.tsx:41]: Is this here mainly to share styles?

File: src/pages/settings_page.tsx [hunk 2]
```diff
@@ -5,14 +5,23 @@ import {Footer} from './footer';
...
     const [churchSelected, setChurchSelected] = useState(settings.church)
     const [supportLogin, setSupportLogin] = useState(false)
 
+    function credentials(): string | undefined {
+        if(settings.credentials) {
+            const user: any = jwt_decode(settings.credentials)
+            return user.name
+        }
+        return "Sign In"
+    }
```
Comment (michael-hewitt) [inline src/pages/settings_page.tsx:23]: Seems slightly overloaded to return the user name or "Sign In".  Perhaps consider naming this function something like `userName()`, have it not return anything if there are no credentials, and then below you can do something like `{userName() ?? "Sign In"}`.

**Review by michael-hewitt — APPROVED** [review]
Looks great!  No need to apply my feedback within this PR - that would cause pain for your other branches.  You can always fix forward (apply the feedback in a future PR).

**Comment by esump** [comment]
This is what it should look like if you're not signed in and the computer remembers you.

Screen Shot 2022-09-14 at 18 12 44

**Comment by esump** [comment]
This is what it should look like signed in

Screen Shot 2022-09-14 at 18 23 53

**Pull request merged**